### ISSUE TITLE
Is this still a place to submit bugs?

### ISSUE THREAD

**Issue by michaldybczak** [kotelnik/plasma-applet-active-window-control#144]
I want to submit a bug but I got confused.

- plasma-applet-active-window-control is no longer developed on githib, it's in phabircator under kde
- There is no plasma-applet-active-window-control on kde bugtracker list so I can't file a bug there
- There are no ways to install fresh plasma-applet-active-window-control. github version is old, AUR git version is thus also old, arch/manjaro repo version is old, no way to find a package on Neon or find any install instructions for recent ubuntus, on kde store ithere is also obsolete version.
- I have a bug but how can I be sure it's not fixed if I can't get any recent version on popular ubuntu or arch distro families or anywhere on that matter?

If plasma-applet-active-window-control is under kde I would expect it to be shipped with Plasma desktop by default, alas it isn't.

I have a issue with certain global menus (firefox and few electron ones) that are visible but not active (Manjaro). In case of firefox they can be make active after few weird actions. I tested it and the issue is not present on system global menu panel and is independent whether it's on system panel or latte panel. I can't install plasma-applet-active-window-control on Neon so I can't really test it there. I would like to open a new bug for this issue but got confused by the changes, lack of current version, unsure where to post. This is in short. I'm not submitting this bug here yet. In this topic I submit only those things:

1. What is the state of plasma-applet-active-window-control project?
2, How can we get up to applet?
3. Where to post bugs?

plasma-applet-active-window-control is one of the crucial elements of my desktop setup and I love it and I would like to help by submitting bugs and forward people to the info how they can install working, up to date version.

**Comment by trmdi** [comment]
It seems that this project has been inactive for a long time.

But I don't know what you mean by saying "There are no ways to install fresh plasma-applet-active-window-control. github version is old, AUR git version is thus also old..." ?

You can still clone the git version from KDE's git and build it, can't you? I'm doing that without any problem.

**Comment by michaldybczak** [comment]
https://phabricator.kde.org/source/plasma-active-window-control/

shows that there are commits, latest was in june 26th

Still, it's not clear what is the status of the project. If it's under KDE so it should either come with plasma desktop by default or be in normal plasma repos - clear and simple. Bugs should be possible to submit on bugtracker. And yet, no official up to date version anywhere aside coping it from Phabricator and installing via script which is also not ideal (no integration with system, no automatic updates, etc.).

People love what AWC can do and ask me how did I get this and I always tell them to get git version as it is the only one that really works but... it shows a symptoms of aging and without updates on github we're left cold with old code and issues. This is a mess.

**Comment by trmdi** [comment]
I found his email in this: https://github.com/kotelnik/plasma-applet-active-window-control/blob/master/package/metadata.desktop

Can you try mailing him ?

**Comment by michaldybczak** [comment]
> But I don't know what you mean by saying "There are no ways to install fresh plasma-applet-active-window-control. github version is old, AUR git version is thus also old..." ?
> You can still clone the git version from KDE's git and build it, can't you? I'm doing that without any problem.

As said above, I can get it directly from phabricator but it's not well integrated with the system. Building things aside pacman/apt database is the last resort.

AUR git version uses git and since there is no update on github, it's obsolete. Manjaro/Arch version is from 2017 so it's also old. The same is with the package on kde store.

> Can you try mailing him ?

Yeah, I will do that but first I will wait a little longer since posts here should land in his mail box anyway. If there will be no contact for longer time I'll try to reach someone from KDE developers to check what is status of the project and who should I contact with.

**Comment by trmdi** [comment]
For the package problem, you should contact the package maintainer. They probably are not the same, so they would update the package. Or you can create a new package yourself.

**Comment by michaldybczak** [comment]
As with all KDE things arojas is the one who takes care about packages and yes, I could look on PKGBUILD and update it, still, I want this package to be in good condition in overall, reliable and easy install for most Linux users.

EDIT: I was mistaken, AUR git version is up to date:

url="https://phabricator.kde.org/source/$_gitpkgname"

So that's out of the way, still now easy way to install it on ubuntus based distros with apt.

That also means, the bug I wanted to file is present in newest version as well. I hoped that this was only a case of obsolete code and with new one it will be gone.

**Comment by trmdi** [comment]
@michaldybczak 
Just curious, could you finally make contact with him?

**Comment by michaldybczak** [comment]
Unfortunately, I got no response here or on mail :(.

**Comment by trmdi** [comment]
I've just opened a new WISHLIST for adding AWC features to the official Global menu widget at: https://bugs.kde.org/show_bug.cgi?id=399308
Please vote for it.

**Comment by michaldybczak** [comment]
Done.

**Comment by trmdi** [comment]
Yeah, there is a new place and a new maintainer for it on KDE Bug Tracker.

https://bugs.kde.org/show_bug.cgi?id=399536
